.github/workflows/wheels.yml

```diff
@@ -0,0 +1,80 @@
+name: Wheel build
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - pyproject.toml
+      - setup.py
+      - .github/workflows/wheels.yml
```

Comment:
Is this worth running on PRs? i.e. is there some nuance of the build process captured here that isn't through a standard soruce install

If so, interested in if there are any other files we'd want to add to this list